proofs.md

```diff
@@ -1,44 +1,55 @@
+- __*Multi-SNARK proof:*__ a block of one or more __*SNARK proofs*__, each proving partition of the total set of challenges to be proved.  
+- __*Merkle inclusion proof:*__ a proof (whose format is unspecified here) that a given leaf block is contained within a merkle tree whose root is the __*commitment*__ associated with the proof.
+- __*commitment:*__ an opaque block of data to which a prover 'commits', enabling subsequent proofs which cannot be validly constructed unless the __*commitment*__ itself was validly constructed. For example: the output of a suitable pseudorandom collision-resistant hash function may serve as a __*commitment*__ to the data which is the preimage of that hash. Publication of the __*commitment*__ proves that the creator was in possession of the preimage at the time the __*commitment*__ was generated.
+- __*prover:*__ the party who generates a proof, in Filecoin it's always the Storage Miner.
+- __*verifier:*__ the party who verifies a proof generated by a __*prover*__, in Filecoin it's a full node.
+- __*sectors count:*__ the number of sectors over which a proof-of-spacetime is performed (`POST_SECTORS_COUNT`).
+
+## Proof-of-Replication Algorithms
 
 __*Filecoin Proof of Replication*__ generates a unique copy (__*sealed sector*__) of a __*sector's*__ __*original data*__, a __*Multi-SNARK proof*__, and a set of __*commitments*__ identifying the __*sealed sector*__ and linking it to the corresponding __*unsealed sector*__.
 
-## Seal
+### PoRep.Seal
 
 `Seal` has the side effect of generating a __*sealed sector*__ from an __*unsealed sector*__,  and returns identifying __*commitments*__ and a __*Multi-SNARK proof*__. The proof returned is a __*Multi-SNARK proof*__.
 
-The commitments are used to verify that the correct __*original data*__ was sealed, and that the correct __*sealed data*__ is the subject of later __*Proofs of Space-Time*__, proving that this data is being stored continuously.
+The commitments are used to verify that the correct __*original data*__ was sealed, and that the correct __*sealed data*__ is the subject of later __*Proofs of Spacetime*__, proving that this data is being stored continuously.
 
-`Seal` operates by performing a slow encoding of the __*unsealed sector*__ — such that it is infeasible for a dishonest prover to computationally regenerate the __*sealed sector*__ quickly enough to satisfy subsequent required __*Proofs of Space-Time*__ — thus ensuring that the __*sealed sector*__ remains manifest as a unique, concrete representation of the __*original data*__.
+`Seal` operates by performing a slow encoding of the __*unsealed sector*__ — such that it is infeasible for a dishonest prover to computationally regenerate the __*sealed sector*__ quickly enough to satisfy subsequent required __*Proofs of Spacetime*__ — thus ensuring that the __*sealed sector*__ remains manifest as a unique, concrete representation of the __*original data*__. 
```

Comment:
`quickly enough to satisfy subsequent required ` => `quickly enough to produce subsequent required` 
I don't believe we can say "satisfy a proof", do we ? (but we can say the proof satisfy the verification checks or stg)